packages/api-client/src/main/tcp/ReconnectingWebsocket.ts

```diff
@@ -0,0 +1,157 @@
+/*
+ * Wire
+ * Copyright (C) 2019 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {TimeUtil} from '@wireapp/commons';
+import logdown from 'logdown';
+import NodeWebSocket = require('ws');
+
+import RWS, {Options} from 'reconnecting-websocket';
+import {CloseEvent, ErrorEvent, Event} from 'reconnecting-websocket/dist/events';
+import * as buffer from '../shims/node/buffer';
+
+export enum CloseEventCode {
+  GOING_AWAY = 1001,
+  NORMAL_CLOSURE = 1000,
+  PROTOCOL_ERROR = 1002,
+  UNSUPPORTED_DATA = 1003,
```

Comment:
Let's sort them by value, like `WEBSOCKET_STATE` does, too.

Reply:
It's sorted by name ~.~

Reply:
I know :wink: